static/app/views/relocation/encryptBackup.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import {Button} from 'sentry/components/button';
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import testableTransition from 'sentry/utils/testableTransition';
+import StepHeading from 'sentry/views/relocation/components/stepHeading';
+
+import {StepProps} from './types';
+
+export function EncryptBackup(props: StepProps) {
+  const code =
+    './sentry-admin.sh export global--encrypt-with /path/to/public/\nkey/file.pub /path/to/encrypted/backup/output/file.tar';
+  return (
+    <Wrapper>
+      <StepHeading step={3}>
+        {t('Create an encrypted back up of current self-hosted instance')}
+      </StepHeading>
+      <motion.div
+        transition={testableTransition()}
+        variants={{
+          initial: {y: 30, opacity: 0},
+          animate: {y: 0, opacity: 1},
+          exit: {opacity: 0},
+        }}
+      >
+        <p>
+          {t(
+            'You’ll need to have the public key saved in the previous step accessible and run the command below in your terminal to ensure success'
+          )}
+        </p>
+        <EncryptCodeSnippet
+          dark
+          language="bash"
+          filename=">_ TERMINAL"
+          hideCopyButton={false}
+        >
+          {code}
+        </EncryptCodeSnippet>
+        <p className="encrypt-help">
+          <b>{t('Understanding the command:')}</b>
+        </p>
+        <p>
+          <mark>{'./sentry-admin.sh'}</mark>
+          {'this is a script present in your self-hosted installation'}
+        </p>
+        <p>
+          <mark>{'/path/to/public/key/file.pub'}</mark>
+          {'path to file you created in the previous step'}
+        </p>
+        <p>
+          <mark>{'/path/to/encrypted/backup/output/file.tar'}</mark>
+          {'file that will be uploaded in the next step'}
+        </p>
+        <ContinueButton size="md" priority="primary" onClick={() => props.onComplete()}>
+          {t('Continue')}
+        </ContinueButton>
+      </motion.div>
+    </Wrapper>
+  );
+}
+
+export default EncryptBackup;
+
+const EncryptCodeSnippet = styled(CodeSnippet)`
+  margin: ${space(2)} 0 ${space(4)};
+  padding: 4px;
+`;
+
+const Wrapper = styled('div')`
+  max-width: 769px;
```

Comment:
These values seem oddly specific. Are we sure this is the standard way to do this on front end. Ex: is this still the right value for mobile screens?

Reply:
Unsure, I used these numbers specifically from the design spec, since it had these values in it